templates/_components/index.html

```diff
@@ -509,6 +807,194 @@ <h2>Skip link</h2>
+              {% #table id="customTable" %}
+                {% #table_head class="bg-slate-200" %}
+                  {% #table_row %}
+                    {% #table_header %}
+                      <div class="flex flex-row gap-2">
+                        Project
+                        <span class="ml-auto">
+                          {% icon_arrows_up_down_mini class="h4 w-4 datatable-icon--no-sort" %}
+                          {% icon_arrow_up_mini class="h4 w-4 datatable-icon--ascending" %}
+                          {% icon_arrow_down_mini class="h4 w-4 datatable-icon--descending" %}
+                        </span>
+                      </div>
+                    {% /table_header %}
+                    {% #table_header %}
+                      <div class="flex flex-row gap-2">
+                        Job requests run
+                        <span class="ml-auto">
+                          {% icon_arrows_up_down_mini class="h4 w-4 datatable-icon--no-sort" %}
+                          {% icon_arrow_up_mini class="h4 w-4 datatable-icon--ascending" %}
+                          {% icon_arrow_down_mini class="h4 w-4 datatable-icon--descending" %}
+                        </span>
+                      </div>
+                    {% /table_header %}
+                    {% #table_header data-type="date" data-format="DD MMM YYYY" %}
+                      <div class="flex flex-row gap-2">
+                        Job first run
+                        <span class="ml-auto">
+                          {% icon_arrows_up_down_mini class="h4 w-4 datatable-icon--no-sort" %}
+                          {% icon_arrow_up_mini class="h4 w-4 datatable-icon--ascending" %}
+                          {% icon_arrow_down_mini class="h4 w-4 datatable-icon--descending" %}
+                        </span>
+                      </div>
+                    {% /table_header %}
+                  {% /table_row %}
+                {% /table_head %}
+
+                {% #table_body %}
+                  {% #table_row class="even:bg-slate-50" %}
+                    {% #table_cell %}
+                      101 - Test project for a table
+                    {% /table_cell %}
+                    {% #table_cell %}
+                      786
+                    {% /table_cell %}
+                    {% #table_cell %}
+                      31 October 2022
+                    {% /table_cell %}
+                  {% /table_row %}
+                {% /table_body %}
+              {% /table %}
+              {% table_pagination has_previous=True has_next=True next_url="#" prev_url="#" %}
+            </div>
+          {% /card %}
+        </div>
+        <pre><code class="language-django">{% verbatim %}{% #card class="w-full" no_container=True %}{% endverbatim %}
+  {% filter force_escape %}<div class="inline-block min-w-full align-middle overflow-x-auto max-w-full">{% endfilter %}
+    {% verbatim %}{% #table id="customTable" %}
```

Comment:
This must have been a complete pain to write, but I really like that you have a non-trivial example here. It would have been easy to skip some of the features, but doing it this way makes it far more useful as a reference for future work.

Reply:
Thanks, I tackled this last so had plenty of practice!